### PR TITLE
Better case handling for system tables

### PR DESCRIPTION
**Pull request by Hydrocharged**

### CODE REVIEW

File: go/libraries/doltcore/sqle/dtables/conflicts_tables.go [hunk 1]
```diff
@@ -43,22 +43,22 @@ type RootSetter interface {
 
 // NewConflictsTable returns a new ConflictsTableTable instance
 func NewConflictsTable(ctx *sql.Context, tblName string, root *doltdb.RootValue, rs RootSetter) (sql.Table, error) {
-	tbl, ok, err := root.GetTable(ctx, tblName)
-
+	var tbl *doltdb.Table
+	var ok bool
+	var err error
+	tbl, tblName, ok, err = root.GetTableInsensitive(ctx, tblName)
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/dtables/conflicts_tables.go:49]: Seems silly to declare all these vars just to reuse one variable here

Comment (zachmu) [reply go/libraries/doltcore/sqle/dtables/conflicts_tables.go:49]: Same comment in all these files

**Review by zachmu — APPROVED** [review]
Needs a bats test or two, otherwise LGTM

File: go/libraries/doltcore/sqle/dtables/conflicts_tables.go (hunk 1 above)
Comment (Hydrocharged) [reply go/libraries/doltcore/sqle/dtables/conflicts_tables.go:49]: This `:=` syntax is not my favorite

**Pull request merged**